CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 ## [4.8.0]
 
 - https://github.com/wazuh/wazuh-packages/releases/tag/v4.8.0
+- Add updated AWS integration files to Solaris 11 [#2408](https://github.com/wazuh/wazuh-packages/pull/2408)
```

Comment:
This is not necessary, changelog is generated automatically on the repository release.